docs/reuse/tutorial/setup.rst

```diff
@@ -1,34 +1,46 @@
 We recommend starting from a clean Ubuntu 22.04 installation. If you don't have
 one available, you can create one using `Multipass`_:
 
-.. collapse:: How to create an Ubuntu 22.04 VM with Multipass
+.. tabs::
 
-    Is Multipass_ already installed and active? Check by running
+    .. tab:: Ubuntu
 
-    .. code-block:: bash
+        Is Multipass_ already installed and active? Check by running
 
-        sudo snap services multipass
+        .. code-block:: bash
 
-    If you see the ``multipass`` service but it isn't "active", then you'll
-    need to run ``sudo snap start multipass``. On the other hand, if you get
-    an error saying ``snap "multipass" not found``, then you must install
-    `Multipass <install-multipass_>`_:
+            sudo snap services multipass
 
-    .. code-block:: bash
+        If you see the ``multipass`` service but it isn't "active", then you'll
```

Comment:
I wonder if we can assume that multipass is active and skip to the multipass installation sentence. A user that manually disabled their multipass snap may already know how to correct this issue.

Reply:
There is the deliberately disabled part where we could skip it, there is the accidentally disabled and the something went wrong on the machine where it could be useful to have this as a check

Reply:
That is true. In my opinion it comes down to how many people are affected by this issue to keep the tutorial concise. 

If it is a small, but significant fraction of users who have accidentally disabled multi pass, we could add a row to the troubleshooting guide at the bottom. Unfortunately we haven't haven't had many people use multipass in the UX sessions to get a indication how common this problem is.

I just disabled my multipass snap to see what happens:

``` bash

:~$ sudo snap disable multipass 
2024-08-30T09:13:30+02:00 INFO Waiting for "snap.multipass.multipassd.service" to stop.
multipass disabled
:~$ snap services
Service                                              Startup   Current   Notes
...
lxd.user-daemon                                      enabled   inactive  socket-activated
multipass.multipassd                                 disabled  inactive  -
snapd-desktop-integration.snapd-desktop-integration  enabled   active    user
:~$ multipass list
Command 'multipass' not found, but can be installed with:
sudo snap install multipass
:~$ sudo snap install multipass
snap "multipass" is already installed, see 'snap help refresh'
```

Reply:
Ok, let's remove this part for now and see if we need it

Reply:
I'm looking at the sentence and it seems to just be explaining the output of the `snap services multipass` command. If that sentence is removed, it might not make as much sense. @cjdcordeiro any thoughts please?

Reply:
@jdkandersson  Yeah, that is true. Is multipass pre-installed on any releases? We could direct the user to install multipass like we do in the `Windows` and `macOS` tabs. 

If multipass was installed on the system beforehand, and the user knows about it, they would probably skip the installation. On the otherhand if multipass is installed and the user does not know it is, `snap install` would return that multipass is already installed and the user would continue forward.

Reply:
I don't think multipass is pre-installed at this point